server/tests/ballot_comparison/snapshots/snap_test_ballot_comparison.py

```diff
@@ -109,39 +109,36 @@
 \r
 ######## ROUNDS ########\r
 Round Number,Contest Name,Targeted?,Sample Size,Risk Limit Met?,P-Value,Start Time,End Time,Audited Votes\r
-1,Contest 1,Targeted,22,No,0.1495325187,DATETIME,DATETIME,Choice 1-1: 10; Choice 1-2: 9\r
-1,Contest 2,Opportunistic,,No,0.1124219565,DATETIME,DATETIME,Choice 2-1: 15; Choice 2-2: 9; Choice 2-3: 8\r
-2,Contest 1,Targeted,41,Yes,0.0013218788,DATETIME,DATETIME,Choice 1-1: 24; Choice 1-2: 17\r
-2,Contest 2,Opportunistic,,Yes,0.0113963098,DATETIME,DATETIME,Choice 2-1: 21; Choice 2-2: 10; Choice 2-3: 13\r
+1,Contest 1,Targeted,22,No,3.9787903602,DATETIME,DATETIME,Choice 1-1: 10; Choice 1-2: 8\r
+1,Contest 2,Opportunistic,,No,2.9913452992,DATETIME,DATETIME,Choice 2-1: 14; Choice 2-2: 8; Choice 2-3: 8\r
+2,Contest 1,Targeted,24,No,4.2908708982,DATETIME,DATETIME,Choice 1-1: 12; Choice 1-2: 7\r
```

Comment:
@umbernhard this seems weird to me - why would the sample size for round 2 go down when the total discrepancies for round 1 went up? I generally try to have this test meet the risk limit in round 2, but it's not happening because the sample size for round 2 is so small that there's no way to balance out the discrepancies from round 1.

Reply:
Hrm. I'm not sure, I'm gonna look into it tomorrow.